src/commands/docker.ts

```diff
@@ -93,7 +92,7 @@ export async function runWorkflowWithProgress(
           // early returns handled within each workflow depending on how far it got
         });
 
-        logger.debug(`running ${workflow.constructor.name} workflow`, { start });
+        logger.debug(`running ${workflow.resourceKind} workflow`, { start });
```

Comment:
Unrelated cleanup after some debugging earlier where we saw `running yg workflow` and realizing the constructor names were minimized for non-dev builds.